src/wkwebview/mod.rs

```diff
@@ -806,10 +808,13 @@ impl InnerWebView {
             run_file_upload_panel as extern "C" fn(&Object, Sel, id, id, id, id),
           );
 
-          ctl.add_method(
-            sel!(webView:requestMediaCapturePermissionForOrigin:initiatedByFrame:type:decisionHandler:),
-            request_media_capture_permission as extern "C" fn(&Object, Sel, id, id, id, id, id),
-          );
+          // https://github.com/tauri-apps/wry/issues/1101
+          if let (14, 0, _) = operating_system_version() {
```

Comment:
Should we add `#[cfg(target_os = "macos")]` check here?

Reply:
yeah it probably should